context.c

```diff
@@ -398,6 +398,52 @@ const struct iio_backend * const iio_backends[] = {
 };
 const unsigned int iio_backends_size = ARRAY_SIZE(iio_backends);
 
+static int iio_context_update_scale_offset(struct iio_context *ctx)
+{
+	const struct iio_attr *attr;
+	struct iio_channel *chn;
+	struct iio_device *dev;
+	unsigned int i, j;
+	int err;
+
+	if (!ctx->ops->read_attr) {
+		/* read_attr callback not implemented - nothing to do. */
```

Comment:
nit: don't think the comment adds much value. The code is very self explanatory...